workers/unity/Packages/io.improbable.gdk.tools/GenerateCode.cs

```diff
@@ -75,15 +75,14 @@ private static bool CanGenerateOnLoad()
         [MenuItem("SpatialOS/Generate code", isValidateFunction: false, MenuPriorities.GenerateCodePriority)]
         private static void GenerateMenu()
         {
-            Debug.Log("Generating code...");
```

Comment:
Any reason for removing this?

Reply:
woops 😅

Reply:
removed it to test something and forgot to add it back